library/modules/bigip_device_info.py

```diff
@@ -12534,7 +12538,15 @@ def address(self):
     @property
     def netmask(self):
         parts = self._values['address'].split('/')
-        return to_netmask(parts[1])
+        if is_valid_ip(parts[0]):
+            addr = ip_address(u'{0}'.format(parts[0]))
+            if addr.version == 4:
```

Comment:
if else statement returns same value; if this is intended than not needed.

Reply:
I didn't even notice that! Thanks!!